CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+See [Adding to this Changelog](#adding-to-this-changelog) section below for details on how to add notes to this file
```

Comment:
it might make sense to cite our [backward compatibility policy](https://docs.solana.com/developing/backwards-compatibility) in this paragraph as well